### DISCUSSION TITLE
Not able expand env variables in Kitty config on MacOS

### DISCUSSION THREAD

**Discussion by bhushan** [kovidgoyal/kitty#5290] [Q&A] [answered]
<img width="1399" alt="Screenshot 2022-07-18 at 10 59 46 PM" src="https://user-images.githubusercontent.com/43483545/179569095-c6591d93-bbda-498c-8239-7f4f1aa35881.png">

If you see in screenshot, I wan to be able to do what is in first two lines in config. I tried multiple things and also checked docs several times, Not able to make it work. please anyone help.

**Comment by page-down** [comment]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.env

> env
>
> Specify the environment variables to be set in all **CHILD** processes.

The configuration option `env` will not set kitty's own environment variables.
You need to make sure that you set the correct environment variables when (or before) starting kitty.

Use the following action to check the environment variables that kitty can see.

https://sw.kovidgoyal.net/kitty/actions/#action-show_kitty_env_vars

```conf
map f1 show_kitty_env_vars
```

**Comment by bhushan** [comment, marked as answer]
How can i make sure that include line works ?

**Reply by page-down** [reply]
For what you want to achieve, I suggest not using environment variables, but include `current-theme.conf` and then make a symlink to the theme's configuration file.

> How can i make sure that include line works ?

For the include line only, you need to make sure the environment variables are set before running kitty. This is not easy under macOS.

If you want to set environment variables for kitty when running via Launchpad, Spotlight, etc., then you can probably use `launchctl setenv`.

Here is the previous discussion:
https://github.com/kovidgoyal/kitty/issues/5033
